ethstore/src/json/hash.rs

```diff
@@ -92,6 +90,12 @@ macro_rules! impl_hash {
 			}
 		}
 
+		impl From<&'static str> for $name {
+			fn from(s: &'static str) -> Self {
+				s.parse().unwrap()
```

Comment:
`.expect("invalid string literal for {}: '{}'", stringify!($name), s)` would be a little more helpful